docs/project_setup_and_build.md

```diff
@@ -0,0 +1,230 @@
+# Project structure and build
+
+## Overview
+
+In this chapter, you will get to know the anatomy of the standard BigFlow project, artifacts, and CLI commands. If you are interested
+in the details of the interface itself, go to [the CLI chapter](./cli.md).
+
+The BigFlow build system packages your processing logic into a Docker image. Thanks to this approach, you can create any
+environment you want for your pipelines. What is more, you don't need to worry about clashes with the Cloud Composer (Airflow) dependencies.
+
+The scheme of a BigFlow artifact looks like this:
+
+![BigFlow artifact scheme](./images/bigflow-artifact.png)
+
+As you can see, your project is turned into a standard Python package (that you can, for example, upload to [pypi](https://pypi.org/) or install locally using `pip`). 
+Next, the package is installed on the Docker image. Finally, there are Airflow DAGs that utilize the produced image.
```

Comment:
the package is closed/sealed into a Docker image with fixed Python version

Reply:
done